Library/include/CSP/Systems/Users/UserSystem.h

```diff
@@ -141,6 +140,7 @@ class CSP_API CSP_NO_DISPOSE UserSystem : public SystemBase
 									 const csp::common::String& Email,
 									 const csp::common::String& Password,
 									 bool ReceiveNewsletter,
+									 bool HasVerifiedAge,
```

Comment:
Should there be an xml comment to describe this parameter as well ?

Reply:
I must have missed that one